src/components/common/HelpSection.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="help-section-component">
+    <div class="help-btn top" @click="helpToggle = !helpToggle">
+      <v-icon color="primary">mdi-help-circle-outline</v-icon>
+      <span v-if="!helpToggle" class="pl-2">{{ header }}</span>
+      <span v-else class="pl-2">Hide Help</span>
+    </div>
+
+    <v-expand-transition v-if="helpToggle">
+      <section class="help-section info-text">
+        <header class="help-header mt-4">
+          <h2>Contact BC Registries</h2>
+        </header>
+
+        <p v-for="(item, index) in items" :key="index" class="mt-4 py-1" v-html="item" />
+
+        <!-- BC Registry Contacts -->
+        <BcRegContacts class="mb-6" :direction="'col'"/>
+
+        <label>Hours of Operation:</label>
+        <p>Monday to Friday, 8:30am - 4:30pm Pacific Time</p>
+      </section>
```

Comment:
No "hide" button?

Reply:
Well, initially there wasn't, but i see the designs have since been updated to include it. 
I promise i triple checked this because i noticed this specifically. 

Looks like i will be putting it in now 😩

Reply:
I laughed a little but I also feel for you. Here:

```
<div class="help-btn bottom" @click="helpToggle = !helpToggle">Hide Help</div>
```

Reply:
hahah its all we can do..

thank you, will insert 😁

Reply:
Oh, and this, too:

https://github.com/bcgov/business-create-ui/blob/299527f5e6b0f353f0b16f0891eb2ce52d547f52/src/components/common/HelpSection.vue#L69

Reply:
Added!